scipy/integrate/tests/test_quadpack.py

```diff
@@ -342,6 +342,7 @@ def simpfunc(z, y, x, t):      # Note order of arguments.
                             (2.,)),
                      2*8/3.0 * (b**4.0 - a**4.0))
 
+    @pytest.mark.xslow
```

Comment:
There are plenty of tests for 2D improper integrals, and there is other coverage of 3D integrals. Both of these functions just call `nquad`, so I think it's safe enough to run these only selectively, like when work is done on the functions (rare).